Automattic/newspack-plugin
# Dashboard: Add Material icons to the different cards and normalize header

### All Submissions:






### Changes proposed in this Pull Request:

I replaced the icons with Material icons in order to match the wizard pages (see #307)

* Remove old svg files
* Replace with material icons (path is stored in `class-dashboard.php`)
* Rework header to match setup wizard
* Add a wrapper to the name and description of the card

__Before:__

![1 before](https://user-images.githubusercontent.com/177929/69053131-a1db4100-0a00-11ea-906e-bbf222db5e81.png)

__After:__

![2 after](https://user-images.githubusercontent.com/177929/69053137-a6075e80-0a00-11ea-8a63-ad54ad3f5960.png)

### How to test the changes in this Pull Request:

1. Switch to this branch and run `npm run build:webpack`
2. Go to the Dashboard
3. Do you see the new icons?

### Other information:
